relay-server/src/extractors/request_meta.rs

```diff
@@ -199,6 +200,9 @@ pub struct RequestMeta<D = PartialDsn> {
     // NOTE: This is internal-only and not exposed to Envelope headers.
     #[serde(skip, default = "Instant::now")]
     start_time: Instant,
+
+    #[serde(default, skip_serializing_if = "ClientHints::is_empty")]
+    client_hints: ClientHints<String>,
```

Comment:
I'd move this a bit upwards and place it next to `user_agent`.

Reply:
right, I was worried there would be serialization issues, but i guess with json it doesnt matter